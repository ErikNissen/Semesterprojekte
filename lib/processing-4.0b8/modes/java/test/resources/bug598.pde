version https://git-lfs.github.com/spec/v1
oid sha256:bf3b267a9e9775a224e3242baef5ec1a261e98acadcca014436803049fe5b781
size 960
