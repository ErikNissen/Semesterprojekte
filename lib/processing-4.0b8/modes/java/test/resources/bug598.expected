version https://git-lfs.github.com/spec/v1
oid sha256:7ed9a9921601deae7cab2202acc293b5f5c40016c85ea4681077d7fac78c2cfd
size 1647
